### ISSUE TITLE
adjust the height of the lightbox? 

### ISSUE THREAD

**Issue by bullishbil** [jsor/lity#239]
Hi, 
I am trying to use Lity to show a content of a site/URL. Is there a way to adjust the height of the initial lightbox to a specific height OR auto adjust the initial lightbox height to show the entire page (so no need to scroll?

Any tips/solutions for this is greatly appreciated. Thanks!

**Comment by naton** [comment]
Yes, you'll need to override `.lity-iframe-container` with another height.
